### PR TITLE
Fix for #199

### PR DESCRIPTION
**Pull request by julesjacobsen**
Sorry about the formatting - I changed the tabs to spaces...

I tested this under load in exomiser running in a web server running 4 simultaneous analysis jobs. Before the fix the number of reported annotated single allele variants would differ wildly. For example:

Single-job:
```
[http-nio-8080-exec-2] INFO  de.charite.compbio.exomiser.core.factories.VariantFactory - Processed 37709 variant records into 37300 single allele variants, 409 are missing annotations, most likely due to non-numeric chromosome designations
```
Multiple jobs (same sample so these ought to have the same numbers as the single job above):
```
[http-nio-8080-exec-11] INFO  de.charite.compbio.exomiser.core.factories.VariantFactory - Processed 37709 variant records into 37188 single allele variants, 521 are missing annotations, most likely due to non-numeric chromosome designations
[http-nio-8080-exec-10] INFO  de.charite.compbio.exomiser.core.factories.VariantFactory - Processed 37709 variant records into 37219 single allele variants, 490 are missing annotations, most likely due to non-numeric chromosome designations
[http-nio-8080-exec-8] INFO  de.charite.compbio.exomiser.core.factories.VariantFactory - Processed 37709 variant records into 37152 single allele variants, 557 are missing annotations, most likely due to non-numeric chromosome designations
[http-nio-8080-exec-7] INFO  de.charite.compbio.exomiser.core.factories.VariantFactory - Processed 37709 variant records into 37218 single allele variants, 491 are missing annotations, most likely due to non-numeric chromosome designations
```
Multiple jobs, post fix:
```
[http-nio-8080-exec-14] INFO  de.charite.compbio.exomiser.core.factories.VariantFactory - Processed 37709 variant records into 37300 single allele variants, 409 are missing annotations, most likely due to non-numeric chromosome designations
[http-nio-8080-exec-11] INFO  de.charite.compbio.exomiser.core.factories.VariantFactory - Processed 37709 variant records into 37300 single allele variants, 409 are missing annotations, most likely due to non-numeric chromosome designations
[http-nio-8080-exec-13] INFO  de.charite.compbio.exomiser.core.factories.VariantFactory - Processed 37709 variant records into 37300 single allele variants, 409 are missing annotations, most likely due to non-numeric chromosome designations
[http-nio-8080-exec-12] INFO  de.charite.compbio.exomiser.core.factories.VariantFactory - Processed 37709 variant records into 37300 single allele variants, 409 are missing annotations, most likely due to non-numeric chromosome designations
```

### CODE REVIEW

File: jannovar-core/src/main/java/de/charite/compbio/jannovar/annotation/AnnotationCollector.java [hunk 1]
```diff
@@ -55,8 +55,8 @@
  *
  * Used for the implementation of VariantAnnotator.
  *
- * @author <a href="mailto:peter.robinson@charite.de">Peter N Robinson</a>
- * @author <a href="mailto:max.schubach@charite.de">Max Schubach</a>
+ * @author Peter N Robinson <peter.robinson@charite.de>
+ * @author Max Schubach <max.schubach@charite.de>
```
Comment (visze) [inline jannovar-core/src/main/java/de/charite/compbio/jannovar/annotation/AnnotationCollector.java]: This should be:
```
* @author <a href="mailto:peter.robinson@charite.de">Peter N Robinson</a>
 * @author <a href="mailto:max.schubach@charite.de">Max Schubach</a>
```
and not `Peter N Robinson <peter.robinson@charite.de> ` (wrong javadoc format)

**Comment by visze** [comment]
It seems to me, that there are conflicts,. 
@julesjacobsen  can you fix them?

**Comment by holtgrewe** [comment]
What's up with this?

**Comment by julesjacobsen** [comment]
Oh yikes. Sorry - I didn't see this lot. Actually I thought you manually made the change - you can see on development that AnnoataionCollector has been removed and its single usage is now a local ArrayList in VariantAnnotator. These were the key changes I made so I think we're ok.

**Pull request closed without merging**